### PR TITLE
Don't log errors on unpacked extensions

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

**Comment by eishay** [comment]
nice :-)

File: packrat/main.js [hunk 1]
```diff
@@ -1099,6 +1099,11 @@ authenticate(function() {
 // Global error logging
 
 function reportError(errMsg, url, lineNo) {
+  api.log('Logging error "%s" in %s line %s', errMsg, url, lineNo);
```
Comment (2is10) [inline packrat/main.js]: Logging → Reporting ?

Comment (2is10) [inline packrat/main.js]: Errors from content scripts will appear in this log twice now, yes?

**Comment by 2is10** [comment]
Thanks. Merge at will.

File: packrat/main.js (hunk 1 above)
Comment (gmethvin) [inline packrat/main.js]: Yeah, I'll change that.

**Pull request merged**